app/controllers/evaluators_controller.rb

```diff
@@ -21,7 +21,7 @@ def create
     if result[:success]
       redirect_to phase_evaluators_path(@phase), notice: result[:message]
     else
-      flash.now[:alert] = result[:message]
+      flash[:alert] = result[:message]
```

Comment:
this one should probably stay as `flash.now` because you are rendering the page directly with `render` instead of redirecting to a different page where the flash would need to be preserved.